pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -102,21 +108,22 @@ type NodeInfo struct {
 }
 
 // New creates an instance of a mon cluster
-func New(context *clusterd.Context, namespace, dataDirHostPath, version string, size int, placement rookalpha.Placement, hostNetwork bool,
+func New(context *clusterd.Context, namespace, dataDirHostPath, version string, size int, allowMultiplePerNode bool, placement rookalpha.Placement, hostNetwork bool,
```

Comment:
Could this take the `MonSpec` as an arg? Then you wouldn't need to have two args for count and allowMultiplePerNode, and in the future there might be other settings as well for the MonSpec

Reply:
Sure, will do.